datacite-doi-commons/src/main/java/no/unit/nva/doi/DataCiteMetadataDtoMapper.java

```diff
@@ -0,0 +1,97 @@
+package no.unit.nva.doi;
+
+import java.net.URI;
+import java.util.List;
+import java.util.stream.Collectors;
+import no.unit.nva.publication.doi.dto.Contributor;
+import no.unit.nva.publication.doi.dto.Publication;
+import no.unit.nva.publication.doi.dto.PublicationDate;
+import no.unit.nva.publication.doi.dto.PublicationType;
+import no.unit.nva.transformer.dto.CreatorDto;
+import no.unit.nva.transformer.dto.DataCiteMetadataDto;
+import no.unit.nva.transformer.dto.IdentifierDto;
+import no.unit.nva.transformer.dto.PublisherDto;
+import no.unit.nva.transformer.dto.ResourceTypeDto;
+import no.unit.nva.transformer.dto.TitleDto;
+
+public final class DataCiteMetadataDtoMapper {
```

Comment:
TODO: Add class purpose

Reply:
Added, probably good idea for @brinxmat to verify https://github.com/BIBSYSDEV/nva-doi-registrar-client/pull/28/commits/31214e01b4eb6682f516f759b2a131cfffa15047